addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -51,13 +51,15 @@ function factory(dependencies) {
          * @returns {mail.thread|undefined}
          */
         _computeThread() {
-            if (this.res_id) {
-                return insert({
-                    id: this.res_id,
-                    model: this.res_model,
-                });
+            const notificationsThreadIds = this.notifications.filter(notification => notification.message && notification.message.originThread).map(notification => notification.message.originThread.id);
```

Comment:
This line is too long, it would be better to split it.

I think the guideline in that case is having the `.filter` and the `.map` each on a new line (with 1 level of indent), similar as in `_useStoreSelectorThreads`.

I don't feel it's the best, but I don't have a better suggestion.